src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.php

```diff
@@ -192,8 +193,9 @@
             ->tag('kernel.event_subscriber')
             ->tag('monolog.logger', ['channel' => 'messenger'])
 
-        ->set('messenger.listener.stop_worker_on_sigterm_signal_listener', StopWorkerOnSigtermSignalListener::class)
```

Comment:
is it a BC break? 🤔

Reply:
isn't that internal service used by framework itself? how can it be BC-break?